release notes/v0.37.0.md

```diff
@@ -0,0 +1,73 @@
+k6 v0.37.0 is here! 🎉 Mainly it contains fixes and ongoing efforts with refactoring.
+
+## New Features!
+
+### Added an option to output k6 logs to a file through --log-output ([#2285](https://github.com/grafana/k6/pull/2285))
+
+This is on top of the supported stdout/stderr and loki instance.
+
+The feature speaks for itself with examples of usage:
+
+```sh
+k6 run --log-output file=./k6.log --logformat json ./examples/stages.js
+```
+
+with a defined minimal log level:
+
+```sh
+k6 run --log-output file=./k6.log,level=info --logformat json ./examples/stages.js
+```
+
+Thanks, @[alyakimenko](https://github.com/alyakimenko) for the contribution!
+
+**Docs**: [Using file output](https://k6.io/docs/using-k6/options/#file)
+
+### Added experimental basic event loops ([#882](https://github.com/grafana/k6/issues/882))
+
+We added basic event loop functionality ([#2228](https://github.com/grafana/k6/pull/2228) and [#2373](https://github.com/grafana/k6/pull/2373)). This is a first step and currently isn't used by anything, but is now available to extensions. In following releases this will be used by k6 and will mean that some APIs will start having callbacks or return Promises. But first it's likely that it will be used in some experimental extensions.
+
+If you are an extension developer please use it and give your feedback. But take into consideration that it's likely that the API may change.
+
+## Extensions
+
+`v0.37.0` finalizes ([#2376](https://github.com/grafana/k6/pull/2376)) the switching of our internal modules to [a new Go/JavaScript module API](https://k6.io/docs/extensions/guides/create-an-extension/#advanced-javascript-extension).  Lastly, the internal gRPC module was refactored ([#2365](https://github.com/grafana/k6/pull/2365)).
```

Comment:
I feel like the `gRPC` sentence is trying to say that this was the last thing to be done on this? I both don't think this is correct (as we will now need to remove the API) but even it it was I don't think it's necessary to be mentioned ... it makes no difference to the end users.
```suggestion
`v0.37.0` finalizes ([#2376](https://github.com/grafana/k6/pull/2376)) the switching of our internal modules to [a new Go/JavaScript module API](https://k6.io/docs/extensions/guides/create-an-extension/#advanced-javascript-extension).
```

Reply:
:thinking: Originally, I wanted to say that it's the last internal module was migrated and I belive it's important to mention because it can be important to the extensions developers as the example of the migration (even if we reference below to it again)

Reply:
```suggestion
`v0.37.0` finalizes ([#2376](https://github.com/grafana/k6/pull/2376)) the switching of our internal modules ([gRPC module](https://github.com/grafana/k6/pull/2365)) to [a new Go/JavaScript module API](https://k6.io/docs/extensions/guides/create-an-extension/#advanced-javascript-extension).
```

@MStoykov  How about that? :thinking: